### ISSUE TITLE
files: do not implicitly change dir upon closing a tab

### ISSUE THREAD

**Issue by haraldschilly** [sagemathinc/cocalc#4442]
1. imagine I've 3 Tabs, A, B, C, all in different directories a, b and c.
2. I'm working with file C and hence my "files" view is in c
3. When I close C, it will switch over to show file B and **also** change the files directory to b.

That's usually not what I want and it's very annoying. Most of the time I want to open a different file in the same directory where C was. It would be much better if closing tab C lets files stay put in directory c.

**Comment by williamstein** [comment]
I disagree because the current behavior is intentional and exactly matches what Emacs does.

I think we should close this ticket and instead have a ticket about creating a  **completely different**  way of showing files that better solves these problems in a modern way (namely, like in VS Code or IDEs or just have a files "editor" like in operating systems).

**Comment by haraldschilly** [comment]
can't we at least make this a setting? I never used emacs and I can't follow the argument. cocalc isn't emacs.

if such a different editor also behaves like emacs, it might be as annoying for me.

**Comment by williamstein** [comment]
> can't we at least make this a setting?

yes, definitely.  There's no harm in that.

**Comment by slel** [comment]
I like the way JupyterLab does it.

**Comment by williamstein** [comment]
Which is?

**Comment by williamstein** [comment]
OK, I'm surprised to say but I'm getting really annoyed with "my" design where the current working directory for the file/search/etc. changes based on the last file tab you click...  

Maybe we should just change things uniformly so that the file tab doesn't impact the working directory at all.  I'm opened to this change...

**Comment by williamstein** [comment]
This is not a bug.  But definitely leaving open, since it should be configurable, and maybe even the current behavior is not the default. 


Also, the side panel "Tabs" does solve this problem -- you can close a file without changing working directories.  Maybe that is a sufficient solution?  I think it is what @slel suggested...
